client-side/src/components/MyButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './MyButton.scss';
+
+import PropTypes from 'prop-types';
+
+import Button from '@mui/material/Button';
+
+
+const MyButton = ({className, label, onClick, size, disabled = false})=>{
```

Comment:
the name of the component should be something like: Button, or GenericButton.
and the same for all places that you used myButton for example: classes.

Reply:
I didn't use Button because it's the name of the component from MUI.
I'll change it to GenericButton.